src/docfx/build/redirection/RedirectionProvider.cs

```diff
@@ -218,11 +217,11 @@ private static IEnumerable<string> ProbeRedirectionFiles(string docsetPath, Repo
             return renameHistory;
         }
 
-        private Dictionary<string, List<FilePath>> GetPublishUrlMap(IEnumerable<FilePath> redirectUrlSources)
+        private Dictionary<string, List<FilePath>> GetPublishUrlMap(IEnumerable<FilePath> redirectUrlSources, OutputUrlType outputUrlType)
         {
             var fileUrls = new ConcurrentBag<(FilePath file, string url)>();
             var allSources = _buildScope.Files.Concat(redirectUrlSources);
-            ParallelUtility.ForEach(_errorLog, allSources, file => fileUrls.Add((file, _documentProvider.GetDocument(file).SiteUrl)));
+            ParallelUtility.ForEach(_errorLog, allSources, file => fileUrls.Add((file, _documentProvider.GetDocsSiteUrl(file, outputUrlType))));
```

Comment:
Is this always `docs`?
```suggestion
            ParallelUtility.ForEach(_errorLog, allSources, file => fileUrls.Add((file, _documentProvider.GetDocsSiteUrl(file, OutputUrlType.Docs))));
```

Reply:
No, this parameter means the current config OutputUrlType, not the target urlType.

---
In reply to: [415204204](https://github.com/dotnet/docfx/pull/5826#discussion_r415204204) [](ancestors = 415204204)